options/locale/locale_en-US.ini

```
@@ -1175,7 +1175,7 @@ commits.commits = Commits
 commits.no_commits = No commits in common. '%s' and '%s' have entirely different histories.
 commits.nothing_to_compare = These branches are equal.
 commits.search = Search commits…
-commits.search.tooltip = You can prefix keywords with "author:", "committer:", "after:", or "before:", e.g. "revert author:Alice before:2019-04-01".
+commits.search.tooltip = You can prefix keywords with "author:", "committer:", "after:", or "before:", e.g. "revert author:Alice before:2019-01-01".
```

```suggestion
commits.search.tooltip = You can prefix keywords with "author:", "committer:", "after:", or "before:", e.g. "revert author:Alice before:2019-01-13".
```
To disambiguate months and days

---

Well, that's the clearly defined ISO standard format: yyyy-mm-dd , I believe nobody would misunderstand it.

If you like, I can take it.